index.js

```diff
@@ -89,14 +83,20 @@ const init = () => {
     chrome.storage = {
       sync: {
         get: (keys, callback) => {
-          callback(localStorage)
+          if (keys === null ){ keys = Object.keys(localStorage) }
+          let ret = {}
+          keys.forEach(key => ret[key] = JSON.parse(localStorage[key]))
+          callback(ret)
```

Comment:
We haven't discussed style yet. On line 93 you have a semicolon but here you don't.

Reply:
How about prettier #20 as a pre-commit hook? I don't care about the setting so feel free to add your own config file